css/main.css

```diff
@@ -298,7 +298,7 @@ span.close:hover{
 }
 .res{
 	float: left;
-	height: 98%;
+	height: 88%;
```

Comment:
what is the problem you are trying to solve?  the trello card doesn't have a lot of details.

Reply:
nvm, i see it.  next question - did you see this fix solve the problem using the dev tools?

Reply:
The height of the buttons that link to the videos are taller than the height of their container

Reply:
This fix makes it both look better and removes the vertical scrolling option (because it now fits properly and therefore doesn't need to give the option to scroll a couple pixels up and down)

Reply:
Sorry your second message didn't load before I responded. Yes, I used chrome's dev tools.